src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -120,6 +120,32 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
             }
         });
 
+        _updateScrollBar = std::make_shared<ThrottledFunc<ScrollBarUpdate>>(
+            [weakThis = get_weak()](const auto& update) {
+                if (auto control{ weakThis.get() })
+                {
+                    control->Dispatcher()
+                        .RunAsync(CoreDispatcherPriority::Normal, [=]() {
+                            if (auto control2{ weakThis.get() })
+                            {
+                                control2->_isInternalScrollBarUpdate = true;
+
+                                auto scrollBar = control2->ScrollBar();
+                                if (update.newValue.has_value())
+                                {
+                                    scrollBar.Value(update.newValue.value());
+                                }
+                                scrollBar.Maximum(update.newMaximum);
+                                scrollBar.Minimum(update.newMinimum);
+                                scrollBar.ViewportSize(update.newViewportSize);
+
+                                control2->_isInternalScrollBarUpdate = false;
+                            }
+                        });
+                }
+            },
+            std::chrono::milliseconds(8));

```

Comment:
nit: I'd give `8` a name, even if it's something dumb like `ScrollBarUpdateInterval`